src/ftest.jl

```diff
@@ -1,3 +1,10 @@
+struct SingleFTestResult
+    dof::Int
+    nulldof::Int
```

Comment:
Actually it would make sense to store `nobs` instead, for consistency with `FTestResult`. It's also nicer for interpretation. Then I'd print the number of observations in the output (like for the existing `ftest`).